enf-reducer/rte-helpers.metta

```diff
@@ -75,6 +75,20 @@
         $current
     )
 )
+
+(= (foldLeftBinary $f $acc $current Nil) $acc)
```

Comment:
Best to have the function have a type and also a comment of what it does. Example:
```scheme
(= foldLeftBinary (-> (-> $t $t $t) $t $t (List $t) $t))
```